source/components/molecules/BackNavigation/BackNavigation.js

```diff
@@ -86,19 +89,16 @@ const BackNavigation = ({
       ) : null}
     </BackNavigationWrapper>
   ) : (
-    <BackNavigationSingleWrapper style={style}>
-      <BackButton
+    <BackNavigationSingleWrapper inSubstep={inSubstep} style={style}>
+      <CloseButton
+        primary={primary}
         colorSchema={colorSchema}
         onStartShouldSetResponder={() => {
           onBack();
         }}
       >
-        <BackButtonIcon
-          name="keyboard-backspace"
-          style={{ transform: [{ rotate: '-90deg' }] }}
-          colorSchema={colorSchema}
-        />
-      </BackButton>
+        <CloseButtonIcon colorSchema={colorSchema} primary={primary} name="close" />
+      </CloseButton>
```

Comment:
We probably want to just change the icon back to the down-arrow here, to follow the Figma design.

Reply:
I couldnt find the down arrow in the design so thats why i removed it. Maybe im looking at some old design or something..

Reply:
Huh, no, it seems like I'm the one remembering an outdated design, because now in the Subform page there is an x. Okay, then disregard my comment and lets follow what is in Figma.

Reply:
Aight :D